examples/gno.land/p/demo/json/utils.gno

```diff
@@ -0,0 +1,70 @@
+package json
+
+func notDigit(c byte) bool {
+	return c < '0' || c > '9'
+}
+
+func h2i(c byte) int {
```

Comment:
if you do `c |= 32`, you can eliminate the middle case `case c >= 'A' && c <= 'F':`

Reply:
Oh, right, just change the 6th bit, then don't have to do the comparison operation. thnaks for pointing out